fosdem2021.md - draft (#383)
3mdeb/news-and-ideas: blog/content/post/2021-01-27-fosdem2021.md

---
title: FOSDEM 2021 – Open Source Firmware BMC and Bootloader devroom
abstract: 'Abstract first sentence.
          Abstract second sentence.
          Abstract third sentence.'
cover: /covers/image-file.png
author: kamila.banecka
layout: post
published: true
date: 2021-01-27
archives: "2021"

tags:
  - fosdem
  - conference
categories:
  - Firmware
  - OS Dev
  - Security

---
### Introduction

This is another year in which, together with Daniel Kiper from Oracle, we have
the pleasure to organize Open Source Firmware, BMC and Bootloader devroom during
FOSDEM 2021 as a continuation of this devroom of the previous edition. We take
part in this event to promote the use of free and open source
software, and show how it can increase security and the quality of the code
itself and give more business freedom outside of the vendors lock-in, from the
point of the firmware level which is our daily bread in 3mdeb. In this blogpost
we will gather all the information about the event and links to the videos from

> the fosdem 2020.

Reviewer: 2021

Author: done

> > FOSDEM is a free event for software developers to meet, share ideas and

Reviewer 2: ```suggestion
> Free and Open source Software Developers' European Meeting (FOSDEM) is a free event for software developers to meet, share ideas and
```

collaborate. Every year, thousands of developers of free and open source
software from all over the world gather at the event in Brussels. In 2021, they
will gather online.

### Why Open source

> Before diving into the technical information around fosdem, let's make a step

Reviewer 2: ```suggestion
Before diving into the technical information around FOSDEM, let's make a step
```
Let's keep consistent naming of the event across the blog post

 Having open-source code base allows to prove the competencies and gain trust
 without any additional audits.
* The source code has no value itself. Most of the products are made by modifying
existing code components, so technology knowledge that allows performing on the
ready-to-use code in the proposed business model brings the essential value.

### Where to link the event?

* To participate, no registration is needed. Check the [schedule](https://fosdem.org/2021/schedule/track/open_source_firmware_bmc_and_bootloader/) and choose the talks you would take part in. We are in [D.firmware](https://fosdem.org/2021/schedule/room/dfirmware/) room.
* Find the live streaming for a D.firmware [here](https://fosdem.org/2021/schedule/streaming/). The page will be updated every few minutes to show you what is currently
scheduled in each room.

### How we are proceeding this year

This online event will take place on 6 & 7 February 2021. Here you will find [the full schedule](https://fosdem.org/2021/schedule/track/open_source_firmware_bmc_and_bootloader/).
The below points will be updated with recordings after the event.

> ##### February 6 2020

Reviewer: @KBanecka 2021

Author: done

> * [Open Source Firmware status on AMD platforms 2021](https://fosdem.org/2021/schedule/event/firmware_osfsoap2/) by [Piotr Król](https://twitter.com/pietrushnic) and [Marek Kasiewicz](https://fosdem.org/2021/schedule/speaker/marek_kasiewicz/)

Reviewer 2: This schedule contains moderators: Marek, Jakub, Norbert and Cezary. They shouldn't be mentioned along with speakers. At least mark them as moderators or remove them. This is caused by the way Pentabarf interpret other roles, which is weird and incorrrect

Author: ok, thank you for mentioning, I will remove moderators

* [Secure boot without UEFI: booting VMs on Power(PC)](https://fosdem.org/2021/schedule/event/firmware_sbwubvop/) by [Daniel Axtens](https://fosdem.org/2021/schedule/speaker/daniel_axtens/), and [Cezary Sobczak](https://fosdem.org/2021/schedule/speaker/cezary_sobczak/)

* [GRUB - Project Status Update](https://fosdem.org/2021/schedule/event/firmware_gpsu/) by [Daniel Kiper](https://fosdem.org/2021/schedule/speaker/daniel_kiper/), and [Norbert Kamiński](https://fosdem.org/2021/schedule/speaker/norbert_kaminski/)

* [Open source PMCI stack implementation for add-in-card manageability.](https://fosdem.org/2021/schedule/event/firmware_ospsifam/) by [sumanthbhatt](https://fosdem.org/2021/schedule/speaker/sumanthbhatt/), and [Cezary Sobczak](https://fosdem.org/2021/schedule/speaker/cezary_sobczak/)

* [Using OpenSPDM in Firmware as a Foundation for Secure Device Communication](https://fosdem.org/2021/schedule/event/firmware_uoifaaffsdc/) by [Jiewen Yao](https://fosdem.org/2021/schedule/speaker/jiewen_yao/), [xiaoyuruan](https://fosdem.org/2021/schedule/speaker/xiaoyuruan/), and [Norbert Kamiński](https://fosdem.org/2021/schedule/speaker/norbert_kaminski/)

> ### 3mdeb on FOSDEM 2020

Reviewer: @KBanecka 2021

Author: done

---
title: FOSDEM 2021 – Open Source Firmware BMC and Bootloader devroom
abstract: 'Thoughts around FOSDEM 2021 and 2020'
cover: /covers/fosdem.png
author: kamila.banecka
layout: post
published: true
date: 2021-02-02
archives: "2021"

tags:
  - fosdem
  - conference
categories:
  - Firmware
  - OS Dev
  - Security

---
### Introduction

This is another year in which, together with [Daniel Kiper](https://fosdem.org/2021/schedule/speaker/daniel_kiper/) from Oracle, we have the pleasure to organize Open Source Firmware, BMC and
Bootloader devroom during FOSDEM 2021 as a continuation of this devroom of the
previous edition. We take part in this event to promote the use of free and open
source software, and show how it can increase security and the quality of the
code itself and give more business freedom outside of the vendors lock-in, from

> the point of the firmware level which is our daily bread in 3mdeb. In this

Reviewer 3: ```suggestion
the point of the firmware level which is our bread and butter in 3mdeb. In this
```

software from all over the world gather at the event in Brussels. In 2021, they
will gather online.

### Why Open source

Before diving into the technical information around FOSDEM, let's make a step
back and refresh the ground knowledge that made us choose the Open Source path
as users, contributors, developers, supporters – OSS community members.

* Open source builds trust in the features and quality of the software. If the
company wants to start developing OS project, it needs to check its quality. The
published code's quality can be easily scanned by having full insight into the
source code.

* Developers are building a visible portfolio by contributing to the Open Source
project. The company that decided to develop a particular project is more eager
to hire contributors with proven expertise.

* The company can get improved performance for a relatively short amount of time.
 Having open-source repository allows to prove the competencies and gain trust
 without any additional audit or checks.

* The source code has no value itself. Most of the products are made by modifying
existing code components, so technology knowledge that allows performing on the

> ready-to-use code in the proposed business model brings the essential value.

Reviewer 3: ```suggestion
ready-to-use code in the proposed business model brings the essential value. There
are some exceptions to that like very innovative and research base solutions or
fully-featured systems, which secrecy may build value that owners do not want to give
to the community using OSS license.
```
